docs/nydusd.md

```diff
@@ -151,6 +151,31 @@ We are working on enabling cloud-hypervisor support for nydus.
 }
 ```
 
+##### Localdisk Backend
```

Comment:
Maybe mark this `Localdisk Backend (Experimental)` before we have supported it in nydusify.

Reply:
> Maybe mark this `Localdisk Backend (Experimental)` before we have supported it in nydusify.

OK, that's good.